include/zephyr/zbus/zbus.h

```diff
@@ -345,13 +380,25 @@ struct zbus_channel_observation {
  * @param[in] _queue_size The notification queue's size.
  * @param[in] _enable The subscriber initial enable state.
  */
-#define ZBUS_SUBSCRIBER_DEFINE_WITH_ENABLE(_name, _queue_size, _enable)                            \
-	K_MSGQ_DEFINE(_zbus_observer_queue_##_name, sizeof(const struct zbus_channel *),           \
-		      _queue_size, sizeof(const struct zbus_channel *));                           \
-	STRUCT_SECTION_ITERABLE(zbus_observer, _name) = {                                          \
-		ZBUS_OBSERVER_NAME_INIT(_name) /* Name field */                                    \
-			.type = ZBUS_OBSERVER_SUBSCRIBER_TYPE,                                     \
-		.enabled = _enable, .queue = &_zbus_observer_queue_##_name}
+#define ZBUS_SUBSCRIBER_DEFINE_WITH_ENABLE(_name, _queue_size, _enable)       \
+	K_MSGQ_DEFINE(_zbus_observer_queue_##_name,                           \
+		sizeof(const struct zbus_channel *),                          \
```

Comment:
It would force breaking the lines. As I am not using the clang-format, I do not see an issue in letting them like this. Do you? I will start facing many compliance issues because you feel uncomfortable with clang-format results for macros. I suggest fixing the clang-format to avoid this kind of problem.

Reply:
> You sure? Looks to me it'd fit alright
> 
> ```c
>         K_MSGQ_DEFINE(_zbus_observer_queue_##_name,                           \
>                       sizeof(const struct zbus_channel *),                    \
>                       _queue_size,                                            \
>                       sizeof(const struct zbus_channel *)                     \
>         );        
> ```
> 
> it's also usually common practice to either do all arguments in line or one per line (or sometimes split them logically if there's an obvious pairing or something), but the rest of the file is already off that so I normally would not point that out.
> 
> Also I don't think that the parenthesis alignment is strictly part of the coding guideline, but I do care about the indentation style being coherent within the file, and that's how the rest of the file does it (which so happen to be coherent with most of the code base).

I thought you were mentioning aligning with `ZBUS_SUBSCRIBER_DEFINE_WITH_ENABLE( `. That part is ok to go. Sorry, sometimes I get confused with the request change point.

Reply:
Done.

Reply:
Thanks, unfortunately the way github show comment context makes it confusing to understand if the comment was on just a line or the whole block. Starting to do more snippets to avoid ambiguity these days but that takes even more time.

Anyway, looks good, thanks for your patience with this, will do another pass later.